src/services/cli-http-client.js

```diff
@@ -14,10 +14,11 @@ const NETWORK_ERROR_CODES = new Set(['ETIMEDOUT', 'ESOCKETTIMEDOUT', 'ECONNABORT
 const STANDARD_HEADERS = ['user-agent', 'accept-charset', 'connection', 'authorization', 'accept', 'content-type'];
 
 class CliRequestClient {
-  constructor(commandName, logger, http) {
+  constructor(commandName, logger, keytarFlag = false, http = null) {
```

Comment:
If we change the position of the existing variable, we need to update everyplace where `http` is used to set at 3rd position like [this](https://github.com/twilio/twilio-cli-core/blob/main/test/services/cli-http-client.test.js#L35).

Reply:
yeah, should be fine as long as defaults are used

Reply:
Defaults are fine, but the case where we used to pass `http` as 3rd argument like this `new CliRequestClient('blah', logger, { defaults: {} })` but with this change ` { defaults: {} }` is assigned to `keytarFlag` instead of `http`.

Reply:
You're right. What I meant was that in this test case, it doesn't matter what value gets assigned to the HTTP as we're checking that it should be environment value. But I will update it to keep it proper. Thank you

Reply:
Suggest moving `keytarFlag = false` as the fourth param in constructor, to allow the Client to remain backward compatible (Or if any current plugin relies on this Client). 

```
constructor(commandName, logger,  http, keytarFlag = false) {}
```

Reply:
The reason I had done it the other way was that I didn't want to introduce a falsy value for http to make the request but looks like it may be better this way. Updated now.